.github/workflows/buf-ci.yaml

```
@@ -0,0 +1,29 @@
+name: Buf CI
```

[nit] 

So the workflow name matches the file name
```suggestion
name: buf-ci
```